ui/intermittent-failures/BugDetailsView.jsx

```diff
@@ -76,7 +80,44 @@ const BugDetailsView = props => {
     {
       Header: 'Log',
       accessor: 'job_id',
-      Cell: props => <BugLogColumn {...props} />,
+      Cell: _props => {
+        const { value, original } = _props;
+        return (
+          <SimpleTooltip
+            text={
+              <React.Fragment>
+                {`${original.lines.length} unexpected-fail${
+                  original.lines.length > 1 ? 's' : ''
```

Comment:
`original.lines.length > 0` => `original.lines.length`

Reply:
I'm not understanding this suggestion. The existing syntax is correct - return an 's' if there's more than one so the text for "1 unexpected fail" is not pluralized.